sagemaker-experiments/mnist-handwritten-digits-classification-experiment.ipynb

```diff
@@ -449,20 +437,103 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "lineage_table.dataframe()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deploy endpoint for the best training-job / trial component\n",
+    "\n",
+    "Now we'll take the best (as sorted) and create an endpoint for it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Pulling best based on sort in the analytics/dataframe so first is best....\n",
+    "best_trial_component_name = trial_component_analytics.dataframe().iloc[0]['TrialComponentName']\n",
+    "best_trial_component = TrialComponent.load(best_trial_component_name)\n",
+    "\n",
+    "model_data = best_trial_component.output_artifacts['SageMaker.ModelArtifact'].value\n",
+    "env = {'hidden_channels': str(int(best_trial_component.parameters['hidden_channels'])), \n",
+    "       'dropout': str(best_trial_component.parameters['dropout']), \n",
+    "       'kernel_size': str(int(best_trial_component.parameters['kernel_size']))}\n",
+    "model = PyTorchModel(\n",
+    "            model_data, \n",
+    "            role, \n",
+    "            './mnist.py', \n",
+    "            env=env, \n",
+    "            sagemaker_session=sagemaker.Session(sagemaker_client=sm),\n",
+    "            framework_version='1.1.0',\n",
+    "            name=best_trial_component.trial_component_name)\n",
+    "\n",
+    "predictor = model.deploy(\n",
+    "    instance_type='ml.m5.xlarge',\n",
+    "    initial_instance_count=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Cleanup\n",
+    "\n",
+    "Once we're doing don't forget to clean up the endpoint to prevent unnecessary billing.\n",
```

Comment:
Please be explicit that Experiments does not cost you.

Reply:
Message calls out "endpoint" already.